core/src/main/scala/org/ensime/core/SemanticHighlighting.scala

```diff
@@ -56,7 +56,9 @@ class SemanticHighlighting(val global: RichPresentationCompiler) extends Compile
           }
           if (sym.hasFlag(ACCESSOR)) {
             val under = sym.accessed
-            if (under.isVariable) {
+            if (sym.hasFlag(LAZY)) {
```

Comment:
I would be tempted to add a comment here stating that the compiler mis-reports it and that lazy can only be a val.